dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -9,4 +9,6 @@ USER root
 # Install ssh for cloning ssh-repositories
 # Install less for handling git diff properly
 RUN yum install -y sudo git bzip2 which bash curl openssh less && \
-    yum -y clean all && rm -rf /var/cache/yum
+    yum -y clean all && rm -rf /var/cache/yum && \
+    echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
```

Comment:
could you please move that to https://github.com/redhat-developer/codeready-workspaces-theia